contracts/src/tests/test_token_locker.cairo

```diff
@@ -101,14 +103,35 @@ mod test_internals {
         lock_list.append(2);
         lock_list.append(3);
         assert(lock_list.len() == 3, 'should have 3 elements');
-        state.remove_user_lock(2, user);
+        state.remove_lock_from_list(2, lock_list);
         lock_list = state.user_locks.read(user);
         assert(lock_list.len() == 2, 'should have 2 elements');
         assert(lock_list.array().unwrap_syscall() == array![1_u128, 3_u128], 'should have 1 and 3');
 
         // Check that the last element is no longer accessible
         assert(lock_list.get(2).unwrap_syscall().is_none(), 'prev len should be none');
     }
+
+    #[test]
+    fn test_remove_token_lock() {
```

Comment:
This test is not useful, as it's just a replication of the test above with another list provided. As we're only testing the internal function and not the behavior of a higher-level function, this doesn't add value

Reply:
Yeah sure, got it